src/ig/IGExporter.ts

```diff
@@ -653,9 +653,15 @@ export class IGExporter {
       menuItem += `${prefixSpaces}</li>${EOL}`;
     } else {
       menuItem += `${prefixSpaces}<li>${EOL}${prefixSpaces}${'  '}`;
-      if (item.url) menuItem += `<a href="${item.url}">`;
+      if (item.url || item.openInNewTab) {
+        menuItem += '<a ';
+        if (item.openInNewTab) menuItem += 'target="_blank" ';
+        if (item.url) menuItem += `href="${item.url}"`;
+        menuItem += '>';
+      }
       menuItem += item.name;
-      if (item.url) menuItem += '</a>';
+      if (item.isExternal) menuItem += '<img src="external.png" style="text-align: baseline"/>';
```

Comment:
I think we need a space before the image tag.  Right now, there is no space between the menu item text and the img tag, so it looks crowded:
<img width="169" alt="image" src="https://user-images.githubusercontent.com/2278253/83802966-b4a22400-a679-11ea-81d9-123c6e8a2e0f.png">

Adding a space will improve that.  I looked at the [Sample IG](https://build.fhir.org/ig/FHIR/sample-ig/) and it puts a space before the image as well.

Reply:
Added the space back!